docs/configuration/features/input_features.md

```diff
@@ -58,6 +58,33 @@ Each input feature can specify its own preprocessing via the `preprocessing` sub
 It's also possible to specify preprocessing rules for all features of a certain type. See
 [Type-Global Preprocessing](../defaults.md#type-global-preprocessing).
 
+## Common Parameters
+
+### Missing Value Strategy
+
+Ludwig allows any input feature to be "missing" in the dataset (at both training and prediction time). How
+missing values are handled by default varies depending on the feature type, but this handling strategy can be configured
+in `preprocessing` section of the config for each feature:
+
+```yaml
+preprocessing:
+    missing_value_strategy: fill_with_mean
+```
```

Comment:
Is this accurate? I would probably specify something like this to make it clear that these cannot be configured at the top level preprocessing parameters.

```
input_features:
    - name: <name>
      type: category
      preprocessing:
           missing_value_strategy: fill_with_mean
```

Reply:
This is in the feature documentation, so I was hoping that's implied. That's how we reference the other params in the section.